rust/connlib/shared/Cargo.toml

```diff
@@ -42,6 +42,7 @@ libc = "0.2"
 dns-lookup = { workspace = true }
 known-folders = "1.1.0"
 snownet = { workspace = true }
+phoenix-channel = { workspace = true }
```

Comment:
This dependency is not ideal but will go away once we delete this implementation of the phoenix-channel client.